main/distributed-programming.md

```diff
@@ -50,37 +58,60 @@ lockdown(); // turns the current realm into an immutable realm
 // calling this function also adds a global Compartment constructor to create compartments
 ```
 
-
-### Compartments and realms
+## Compartments and realms
 
 JavaScript code runs in the context of a [`Realm`](https://www.ecma-international.org/ecma-262/10.0/index.html#sec-code-realms). A realm is composed of the set of *primordials* (objects and functions of the standard library like `Array.prototype.push`) and of a global object. In a web browser, an iframe is a realm. In Node.js, a Node process is a realm.
 
 For historical reasons, the ECMAScript specification requires the *primordials*
-to be mutable (`Array.prototype.push = yourFunction` is valid ECMAScript but not
-recommended). SES makes it possible to turn the current realm into an
-**immutable realm**, that is a realm within which the primordials are deeply
-frozen. It also allows programs to create **Compartments**.
+to be mutable (`Array.prototype.push = yourFunction` is valid
+ECMAScript but not recommended). SES makes it possible to turn the
+current realm into an **immutable realm**, that is a realm within
+which the primordials are deeply frozen. It also allows programs to
+create **Compartments**. 
 
 Compartments are "mini-realms". They have their own dedicated global object and environment, but they inherit the primordials from their parent realm.
 
-Agoric "deploy scripts" and smart contract code run in an immutable realm with compartments providing just enough authority to create useful and secure contracts (but not enough authority to do anything unintended or harmful to the participants of the smart contract)
+Agoric deploy scripts and smart contract code run in an immutable
+realm with compartments providing just enough authority to create
+useful and secure contracts (but not enough authority to do anything
+unintended or harmful to the participants of the smart contract). 
+This means you cannot send email, write to disk, or visit arbitrary web
+pages, etc. from code inside a contract.
```

Comment:
code cannot do ... unless it was started by a process that had authority to do so, and explicitly passed that authority in.